glusterd2/commands/volumes/commands.go

```diff
@@ -32,6 +32,18 @@ func (c *Command) Routes() route.Routes {
 			Pattern:     "/volumes/{volname}/options",
 			Version:     1,
 			HandlerFunc: volumeOptionsHandler},
+		route.Route{
+			Name:        "VolumeProfileTunables",
+			Method:      "GET",
+			Pattern:     "/volumes/options/{profilename}",
```

Comment:
As it's inexpensive to `GET` all group options at once from etcd, you can simply provide that API which returns everything instead of just one.